src/popover.scss

```diff
@@ -188,14 +192,30 @@ $block: #{$fd-namespace}-popover;
     }
 
     &--no-arrow {
-      box-shadow: $fd-popover-box-shadow-no-arrow;
+      box-shadow: none;
```

Comment:
I'm not sure about this. This is not a double shadow. The specs define different box-shadows when the Popover has an arrow and when it doesn't. 
<img width="538" alt="Screen Shot 2020-06-29 at 4 31 05 PM" src="https://user-images.githubusercontent.com/39598672/86053213-288bde00-ba26-11ea-88b3-69c2e1d22764.png">

Basically `$fd-popover-box-shadow-no-arrow` is overwriting `$fd-popover-box-shadow`